src/events/messages/rawMessageDeleteBulkStarboard.ts

```diff
@@ -0,0 +1,48 @@
+import { DbSet, GuildSettings } from '@lib/database';
+import { Events } from '@lib/types/Enums';
+import { api } from '@utils/Models/Api';
+import { GatewayMessageDeleteBulkDispatch } from 'discord-api-types/v6';
+import { DiscordAPIError, Guild } from 'discord.js';
+import { Event, EventStore } from 'klasa';
+
+export default class extends Event {
+	public constructor(store: EventStore, file: string[], directory: string) {
+		super(store, file, directory, { name: Events.RawMessageDelete, emitter: store.client.ws });
```

Comment:
use discord-api-types event enum

Reply:
Actually, that's a custom one, which processes the guild beforehand, named as the event's name is `'rawMessageDelete'`:

![image](https://user-images.githubusercontent.com/24852502/98996744-aa8f2600-2533-11eb-9c09-b83a6854942a.png)

Reply:
Yet you listen to the ws manager.